vendor/github.com/google/cadvisor/manager/watcher/raw/raw.go

```diff
@@ -110,6 +110,11 @@ func (self *rawContainerWatcher) Stop() error {
 // Watches the specified directory and all subdirectories. Returns whether the path was
 // already being watched and an error (if any).
 func (self *rawContainerWatcher) watchDirectory(dir string, containerName string) (bool, error) {
+	// Don't watch .mount cgroups because they never have containers as sub-cgroups.  A single container
+	// can have many .mount cgroups associated with it which can quickly exhaust the inotify watches on a node.
+	if strings.HasSuffix(containerName, ".mount") {
```

Comment:
Sorry for commenting late but should this also skip `.scope` this issue:

https://github.com/kubernetes/kubernetes/issues/64137

is due to some older version systemd leaving dangling scope cgroups around which cadvisor then watches, skipping `.scope` will help alleviate this

Reply:
I can open a PR on cadvisor if this is deemed useful